### PR TITLE
add soundcloud integration

### PR DESCRIPTION
**Pull request by albertomejia92**
### As a user I can search tracks on soundcloud
[Trello](https://trello.com/c/yHPvNTNC)

### CODE REVIEW

File: app/views/wellcome/index.html.erb [hunk 1]
```diff
@@ -1,9 +1,9 @@
- <form action="">
+ <form action="#" onSubmit="soundcloud();">
```
Comment (Jucerweb) [inline app/views/wellcome/index.html.erb:1]: Why did you add a `#` in action form ?

Comment (albertomejia92) [reply app/views/wellcome/index.html.erb:1]: Because 'action' is not link to any page.

Comment (Jucerweb) [reply app/views/wellcome/index.html.erb:1]: Ok
